doc/api/vm.md

```diff
@@ -171,6 +171,10 @@ const contextifiedSandbox = vm.createContext({ secret: 42 });
     to initialize the `import.meta`. This function has the signature `(meta,
     module)`, where `meta` is the `import.meta` object in the `Module`, and
     `module` is this `vm.SourceTextModule` object.
+  * `importModuleDynamically` {Function} Called during evaluation of this
+    module when `import()` is called.
+    * `specifier` {string} Specifier passed to `import()`.
+    * `module`. {vm.SourceTextModule} This `vm.SourceTextModule` instance.
```

Comment:
Nit: the period between `` `module` `` and `{vm.SourceTextModule}` seems not traditional in this place.